assets/css/stylesheet.css

```diff
@@ -223,7 +223,20 @@ a:hover {
 
 /* Modal */
 #content .featured .content-to-populate-in-modal {
-  display: none;
+  width:500px;
+  margin: 100px auto;
+  background-color: #fff;
+  border:1px solid #000;
+  padding:15px;
+  text-align: center;
+  box-shadow: 0px 0px 8px 8px rgba(255,255,255,1);
+  text-shadow: 0px 0px 10px rgba(255, 255, 255, 0.9);
+  transform: scale(0.9);
+  opacity: 0.9;
```

Comment:
You're right, it does nothing. I left it just as an alternative for styling the modal pop-up window. I didn't know which way is more preferable.